ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewState.kt

```diff
@@ -10,3 +12,9 @@ internal sealed class PaywallViewState {
         val selectedPackage: TemplateConfiguration.PackageInfo,
     ) : PaywallViewState()
 }
+
+internal val PaywallViewState.Loaded.selectedLocalization: ProcessedLocalizedConfiguration
+    get() = selectedPackage.localization
+
+internal val PaywallViewState.Loaded.currentColors: TemplateConfiguration.Colors
+    @Composable get() = templateConfiguration.getCurrentColors()
```

Comment:
Can you add the `@ReadOnlyComposable` annotation as well? That allows to do some extra performance improvements and I think it should be possible here.

Reply:
Done!